package.json

```diff
@@ -138,12 +130,11 @@
     "prettier": "^2.2.1",
     "recast": "^0.20.4",
     "rimraf": "^3.0.2",
-    "rollup": "^2.40.0",
```

Comment:
We used Rollup to building static assets. Static assets was a dirty workaround to serve rulesets in Studio and tests in Spectral.
Luckily, we no longer need these static assets as we're all based on CommonJS now (ES Modules soonish, as a part of monorepo).